apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -768,29 +768,96 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
            end)
   end
 
+  describe "available piggybacks" do
```

Comment:
more test cases needed (remember to always think about the assertions "gist" and focus on asserting a specific behavior, so that your `asserts` are simple and small):
1/ are varying owners handled well
2/ transaction without outputs handled well?
3/ challenged IFEs emit the same piggybacks as canonical ones